app/scripts/views/components/scatterPlot/scatterPlot.js

```diff
@@ -9,6 +9,17 @@
     var chartId_;
     var data_;
     var opts_;
+    var getQtipString = function(_data) {
+      var toReturn = 'Cancer Study:' + _data.study_id + '<br>Sample Id: ' +
+        _data.sample_id + '<br>CNA fraction: ';
+      if (isNaN(_data.cna_fraction)) {
```

Comment:
This is what the current cbioportal has. Cases with 'NA' cna or mutation count data are not shown.